integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/RunPerformanceCucumberTest.java

```diff
@@ -0,0 +1,15 @@
+package com.redhat.service.smartevents.integration.tests;
+
+import org.junit.platform.suite.api.ConfigurationParameter;
+import org.junit.platform.suite.api.IncludeEngines;
+import org.junit.platform.suite.api.SelectClasspathResource;
+import org.junit.platform.suite.api.Suite;
+
+import static io.cucumber.junit.platform.engine.Constants.PLUGIN_PROPERTY_NAME;
+
+@Suite
+@IncludeEngines("cucumber")
+@SelectClasspathResource("performance-features")
+@ConfigurationParameter(key = PLUGIN_PROPERTY_NAME, value = "com.redhat.service.smartevents.integration.tests.cucumber.TestCaseFileLogger,com.epam.reportportal.cucumber.ScenarioReporter")
+public class RunPerformanceCucumberTest {
```

Comment:
Sonar report this class as problematic, as it is empty.

Could you please update `integration-tests/pom.xml` 

from 
`<sonar.exclusion>**/RunCucumberTest.java</sonar.exclusion>`

to
`<sonar.exclusions>**/RunCucumberTest.java,**/RunPerformanceCucumberTest.java</sonar.exclusions>`

Reply:
Thanks for the suggestion!
Done